openapi/components/schemas/history/200-get-sent-messages-request-event.yaml

```diff
@@ -0,0 +1,15 @@
+type: object
+properties:
+  messagesRequestEvents:
+    type: array
+    items:
+      type: object
+      properties:
+        friendshipId:
+          type: string
```

Comment:
I used a `string` because the type `uuid` is not supported. I'm not sure if we need to take any action regarding this.

Reply:
try using:
```
type: string
format: uuid
```

See doc at https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.1.md#data-types